LICENSE

```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2014, Jens Rantil
```

Comment:
This license file should actually be kept. I was mistaken when I asked that it be deleted.